proposals/stage-1/PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -18,100 +18,140 @@ Projects must be proposed via GitHub. Project proposals submitted to the _$FOUND
+
+To be considered for Growth Stage, the project must meet the At Large requirements as well as the following:
+
+ * Development of a growth plan, to be done in conjunction with their project mentor(s) at the CPC.
+ * Document that it is being used successfully in production by at least two independent end users which, in the CPC’s judgement, are of adequate quality and scope.
  * Demonstrate a substantial ongoing flow of commits and merged contributions.
+ * Demonstrate that the current level of community participation is sufficient to meet the goals outlined in the growth plan.
  * Since these metrics can vary significantly depending on the type, scope and size of a project, the CPC has final judgement over the level of activity that is adequate to meet these criteria.
- * Receive a two-thirds supermajority vote of the CPC to move to Incubation Stage. 
+ * Receive a two-thirds supermajority vote of the CPC to move to Growth Stage. 
 
-### Top Level/Graduate Stage*
+### Impact Stage (formerly 'Top-Level')
 **Definition**
 
+The Impact Stage is for projects that have reached their growth goals and are now on a sustaining cycle of development, maintainence, and long-term support. These projects have reached a   
+
 **Examples**
 
+1. Projects that have publicly documented release cycles and plans for LTS.
+1. Projects that have themselves become platforms for other projects.
+1. Projects that are able to attract a healthy number of committers on the basis of its production usefulness (not simply 'developer popularity').
+1. Projects that have several, high-profile or well known end-user implementations.
+
 **Expectations**
 
+Impact Stage projects are expected to participate actively in CPC proceedings, and as such have a binding vote on CPC matters requiring a formal vote, such as the election of a CPC Director. They receive ongoing financial and marketing support from the Foundation, and are expected to cross promote the foundation along with their activities. 
+
 **Acceptance Criteria**
-To graduate from Sandbox or Incubating status, or for a new project to join as a Top-Level project, a project must meet the Incubating stage criteria plus:
 
- * Have committers from at least two organizations.
+To graduate from At Large or Growth status, or for a new project to join as an Impact project, a project must meet the Growth stage criteria plus:
+
+ * Have a defined governing body of at least 5 or more members (owners and core maintainers), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer. 
+ * Have a documented and publicly accessible description of the project's governance, decision-making, and release processes.
+ * Have a healthy number of committers from at least two organizations. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
```

Comment:
IMO we really need explicit numbers here

Reply:
I wonder about this. I was thinking maybe a ratio would work well, but is putting a number or ratio on this requirement just asking to be "gamed" when it isn't satisfied? Perhaps being vague gets us by using the "honor system"?

Reply:
I'd say the CPC vote is really the final acceptance. CPC members will want some guidelines to know what they should be looking for. If they felt like the project was meeting the hard numbers but not the spirit they don't need to approve?

Reply:
@boneskull @jorydotcom is there any existing numbers or consensus on what is "a healthy number" that we can re-use.  If so I'd say lets add that.  If not them my suggestion would be to leave as is for now and keep an action item to figure out what a good number is and plan to add that the doc at a later time.